test/distributed/cases/dml/show/show.test

```diff
@@ -242,3 +242,31 @@ show columns from t3;
 drop table t3;
 
 drop database test;
+
+-- show grants for role
+create role newrole;
+grant all on table *.* to newrole;
+grant create table, drop table on database *.* to newrole;
+create user newuser identified by 'passwd';
+grant newrole to newuser;
+show grants for 'newuser'@'localhost';
+drop user newuser;
+drop role newrole;
+
+-- show index test
+-- @bvt:issue#8392
+drop table if exists index01;
+create table index01(s varchar(20), id int);
```

Comment:
plz add unique index secondary ddl like ,and about mutli key:
create table index_12 (col1 bigint auto_increment primary key,col2 date,col3 date,unique key d1(col2),key d2(col3));
create table index_table_02 (col1 bigint primary key,col2 varchar(25) not null,col3 float default 87.01,col4 int,unique key num_phone(col2),key num_id(col4));